src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -81,26 +81,31 @@ public void SarifErrorListItem_WhenResultRefersToNonExistentRule_ContainsBlankMe
             var result = new Result
             {
                 RuleId = "TST0001",
+                RuleMessageId = "nonExistentMessageId"
             };
 
-            var run = new Run();
+            var run = new Run
+            {
+                Resources = new CodeAnalysis.Sarif.Resources() { }
+            };
 
             var item = MakeErrorListItem(run, result);
 
             item.Message.Should().Be(string.Empty);
         }
 
         [Fact]
-        public void SarifErrorListItem_WhenResultRefersToRuleWithNoMessageStrings_ContainsBlankMessage()
+        public void SarifErrorListItem_WhenResultRefersToRuleWithNoMessageFormats_ContainsBlankMessage()
```

Comment:
Why "message formats"? There isn't anything in the spec anymore called a "message format". All message strings can have `{n}` placeholders now, so "message strings" is a good name.

Reply:
(Here and wherever you did a similar name change.)

Reply:
Possible brain fart.